IOMC/EventVertexGenerators/python/VtxSmearedParameters_cfi.py

```diff
@@ -624,6 +624,32 @@
     Z0 = cms.double(0.6707)
 )
 
+# From fit to 5 TeV pPb data
+Realistic5TeVPACollision2016VtxSmearingParameters = cms.PSet(
+    Phi = cms.double(0.0),
+    BetaStar = cms.double(1100.0),
+    Emittance = cms.double(6.75e-08),
+    Alpha = cms.double(0.0),
+    SigmaZ = cms.double(6.4891),
+    TimeOffset = cms.double(0.0),
+    X0 = cms.double(0.0889),
+    Y0 = cms.double(0.1820),
+    Z0 = cms.double(1.6066)    
```

Comment:
```
-----------------------------------------------------
              Beam Spot Data

 Beam type    = 2
       X0     = 0.0889243 +/- 1.71057e-05 [cm]
       Y0     = 0.181999 +/- 1.70107e-05 [cm]
       Z0     = 1.27369 +/- 0.0942173 [cm]
 Sigma Z0     = 6.81368 +/- 0.0666215 [cm]
 dxdz         = -1.28221e-06 +/- 2.20256e-06 [radians]
 dydz         = 1.96211e-06 +/- 2.18636e-06 [radians]
 Beam Width X = 0.00598025 +/- 6.09373e-05 [cm]
 Beam Width Y = 0.00608967 +/- 6.09373e-05 [cm]
 Emittance X  = 0 [cm]
 Emittance Y  = 0 [cm]
 Beta star    = 0 [cm]
-----------------------------------------------------
```
also for [BeamSpotObjects_Realistic5TeVPACollision2016_v1](https://cms-conddb.cern.ch/cmsDbBrowser/list/Prod/tags/BeamSpotObjects_Realistic5TeVPACollision2016_v1), the fit value of  SigmaZ0 seems to be a bit overestimated.